### PR TITLE
[APPTS-13854] Indicate private key format, improve pem file naming in examples

### PR DESCRIPTION
**Pull request by pkclark**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Addresses issue raised in https://jira.axway.com/browse/APPTS-13854 and improves the provided example file names.

## Deploy preview link

Revised section is here: http://pr-392.amplify-central.opendocs-builder.pcloud.axway.int/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url/#generate-a-tls-certificate

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | a3cc705 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-392.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by cbordier — APPROVED** [review]
I approve the changes

File: content/en/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url.md [hunk 1]
```diff
@@ -41,20 +41,20 @@ It is required that you use a certificate authority to sign your certificate. Se
...
 # windows
-copy *.pem certificate-key.pem
+copy *.pem full-chain.pem
 
 # Linux
-cat certificate.pem key.pem > certificate-key.pem
+cat certificate.pem private-key.pem > full-chain.pem
 ```
 
 ### Step 3 - Upload the `certificate-key.pem` file into the Marketplace Settings
```
Comment (cbordier) [inline content/en/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url.md:60]: @pkclark should we change that to the full-chain.pem?

**Review by lbadenhop — APPROVED** [review]

File: content/en/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url.md (hunk 1 above)
Comment (lbadenhop) [reply content/en/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url.md:60]: @pkclark, I'll merge once you respond to Cyrille's comment.

Comment (pkclark) [reply content/en/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url.md:60]: @cbordier sure should. One sec.

Comment (pkclark) [reply content/en/docs/manage_marketplace/customize_marketplace/marketplace_vanity_url.md]: @lbadenhop two missed refs fixed in https://github.com/Axway/amplify-central/pull/392/commits/c83ebd0510e06f6516374e68e73909d3ccbd33a0

**Pull request merged**